File: management-service/src/main/java/com/stackroute/model/Event.java
package com.stackroute.model;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;
import org.springframework.data.annotation.Id;
import org.springframework.data.mongodb.core.mapping.Document;

import javax.validation.constraints.NotNull;
import java.math.BigDecimal;
import java.util.List;

@Document
@Data
@NoArgsConstructor
@AllArgsConstructor
public class Event {
    @Id
    private String eventId;
    @NotNull
    private String title;
    @NotNull
    private String userEmailId;
    @NotNull
    private String userName;
    @NotNull
    private String eventDescription;
    private List<String> artists;
    private List<String> genre;
    private List<String> languages;
    @NotNull
    private EventTiming eventTimings;
    private List<String> posters;
    private Venue venue;
    private BigDecimal revenueGenerated;
    private int ticketsSold;
    private int totalSeats;
    private List<Seat> seats;
    private int likes;
    private List<String> emailOfUsersLikedEvent;
}

File: product-webapp/angularbase/carnivry/src/app/model/event.ts
import { EventTiming } from "./event-timing";
import { Seat } from "./seat";
import { Venue } from "./venue";

export class Event {
    public eventId: string;
    public title: string;
    public userName:string;
    public userEmailId: string;
    public eventDescription: string;
    public artists: string[];
    public genre: string[];
    public languages: string[];
    public eventTimings: EventTiming;
    public posters: string[];
    public venue: Venue;
    public revenueGenerated: number;
    public ticketsSold: number;
    public totalSeats: number;
    public seats: Seat[];
    public likes: number;
    public emailOfUsersLikedEvent: string[];
    
}
